CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Add NavBar component to frontend

Fixes #90 

Summary:
- Current UI does not have a navbar, users may find it difficult to navigate user-related features
- Adds a navbar containing title, settings icon and logout icon
- Adds tooltip to icon buttons to improve usability
- Configured navbar to show only after logging in - i.e main pages of the application
- Move original logout and change password related code into `NavBar.js`
- Remove unnecessary code

To Test:
1. Start `user-service` and `frontend`
2. After logging in, you should be able to see the nav bar:
![image](https://user-images.githubusercontent.com/65227564/194918015-64960eb5-ff4d-444e-a859-2e2f9f5f60eb.png)

The icons have `tooltips` to improve usability
![image](https://user-images.githubusercontent.com/65227564/194918696-b3ec3119-bf28-4e84-91a4-12b1a3a8b4d6.png)

3. Click on the settings icon, select `Change Password` and you will be directed to the change password page
4. Click on logout icon and you will be logged out.
